interface/src/instruction.rs

```diff
@@ -44,7 +45,9 @@ pub enum StakeInstruction {
     /// [`Authorized`] carries pubkeys that must sign staker transactions
     /// and withdrawer transactions; [`Lockup`] carries information about
     /// withdrawal restrictions.
-    Initialize(Authorized, Lockup),
+    #[codama(account(name = "stake", writable, docs = "Uninitialized stake account"))]
+    #[codama(account(name = "rent_sysvar", docs = "Rent sysvar"))]
+    Initialize { arg0: Authorized, arg1: Lockup },
```

Comment:
I think we need codama support for the tuple variant as this is a breaking change

Reply:
PRed upstream at https://github.com/codama-idl/codama-rs/pull/73